### DISCUSSION TITLE
How to apply a Color theme file from the repository?

### DISCUSSION THREAD

**Discussion by saskford** [mriscoc/Ender3V2S1#83] [Q&A] [answered]
I am confused on how to use the color theme files. It says to download the g-code file and print it. When I download the file I get lines of code. How do you print it?

**Comment by mriscoc** [comment, marked as answer]
Right click, Save link as...
Then copy the file to your µSD card and try to print it.

**Reply by saskford** [reply]
Thank you very much, I would never have figured that out.
